src/platform/forms-system/test/js/widgets/DateWidget.unit.spec.jsx

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import { expect } from 'chai';
 import SkinDeep from 'skin-deep';
+import { render } from '@testing-library/react';
```

Comment:
`skin-deep` hasn't been updated in three years, and it has missing documentation. I couldn't find a way to check attributes with it, so I wrote the test using RTL.